extensions/wikia/FacebookClient/scripts/preferences.js

```diff
@@ -134,6 +144,42 @@
 			window.GlobalNotification.show(msg, 'error');
 		}
 
+		function facebookError() {
+			$(document).on('tab-fbconnect-prefstext-complete', function () {
+				$('.tab-fbconnect-prefstext').on('click', function () {
+					var $tabContentContainer = $('#mw-prefsection-fbconnect-prefstext');
+
+					// Disable everything within the tab
+					$tabContentContainer
+						.find('input, button')
```

Comment:
This seems unnecessary. There's no form elements on this page and there likely will never be.